test/PriceOracleTest.js

```diff
@@ -15,15 +15,16 @@ describe("PriceOracleProxyUSD", function () {
         this.alice = this.signers[0];
 
         // Contract Factory
-        this.priceOracleCF = await ethers.getContractFactory("PriceOracleProxyUSD")
-        this.JCollateralCapErc20Delegate = await ethers.getContractFactory("JCollateralCapErc20Delegate")
-        this.JErc20 = await ethers.getContractFactory("JErc20")
+        this.PriceOracleCF = await ethers.getContractFactory("PriceOracleProxyUSD")
+        this.JCollateralCapErc20DelegateCF = await ethers.getContractFactory("JCollateralCapErc20Delegate")
+        this.JErc20CF = await ethers.getContractFactory("JErc20")
 
         // Tokens
-        this.joe = await this.JCollateralCapErc20Delegate.attach(JOE_ADDRESS)
-        this.xJoe = await this.JCollateralCapErc20Delegate.attach(XJOE_ADDRESS)
-        this.jXJoe = await this.JCollateralCapErc20Delegate.attach(JXJOE_ADDRESS)
-        this.oracleOld = await this.priceOracleCF.attach(PRICEORACLE_OLD_ADRESS)
+        this.joe = await this.JErc20CF.attach(JOE_ADDRESS)
+        // xJoe is also an ERC20, so we can use the JErc20CF.
```

Comment:
Move this comment to above line 23 and change to:
`// We cast to JOE and xJOE to JErc20 because it is also an ERC20`